www/pages/brand-assets.tsx

```diff
@@ -35,7 +53,7 @@ const Index = () => {
             <div className="grid grid-cols-12 border dark:border-gray-600">
               <div className="col-span-12 lg:col-span-5">
                 <img
-                  src={`${basePath}/brand-assets/logo-preview.jpg`}
+                  src={`${router}/brand-assets/logo-preview.jpg`}
```

Comment:
this looks like it should be `${router.basePath}`

Reply:
its possible you can remove this anyway, since we don't use the basePath, I think its undefined